settings.yaml.template

```diff
@@ -109,3 +109,20 @@ AWS:
             # CloudFormations names that would be skipped from cleanup
             STACK_LIST: []
         IMAGES: []
+PODMAN:
+    AUTH:
+        HOSTNAME:
+        USERNAME: root
+        SSH_PORT: 22
+    CRITERIA:
+        CONTAINER:
+            # Container Prepended Test
+            NAME_PREPEND: 'satci'
```

Comment:
It should be prefix. At least in case of podman container cleanup

Reply:
Prefix and prepend are synonyms ? But updated.